exercises/spiral-matrix/canonical-data.json

```diff
@@ -52,10 +52,10 @@
         "size": 4
       },
       "expected": [
-        [ 1,  2,  3, 4],
+        [1, 2, 3, 4],
```

Comment:
while I regret that we are losing the padding, I do not think it is important enough to preserve. I do not think losing the padding is going to cause trouble understanding this expected output.